frontend/src/metabase/visualizations/lib/y_axis_split.js

```diff
@@ -0,0 +1,114 @@
+import d3 from "d3";
+import _ from "underscore";
+
+import { isStacked } from "./renderer_utils";
+
+export default function getYAxisSplit(series, settings) {
+  const datas = series.map(s => s.data.rows);
+  const yExtents = datas.map(d => d3.extent(d.map(row => row[1])));
+
+  const cardType = series[0].card.display;
+
+  const seriesAxis = series.map(single => settings.series(single)["axis"]);
+  const left = [];
+  const right = [];
+  const auto = [];
+  for (const [index, axis] of seriesAxis.entries()) {
+    if (axis === "left") {
+      left.push(index);
+    } else if (axis === "right") {
+      right.push(index);
+    } else {
+      auto.push(index);
+    }
+  }
+
+  // don't auto-split if the metric columns are all identical, i.e. it's a breakout multiseries
+  const hasDifferentYAxisColumns =
+    _.uniq(series.map(s => JSON.stringify(s.data.cols[1]))).length > 1;
+  if (
+    cardType !== "scatter" &&
+    cardType !== "funnel" &&
```

Comment:
I tried preserve the logic here, but might have missed something. In `LineAreaBarRenderer`, we have access to `isScalarSeries`, which is passed as a prop. It looks like that is only ever true for funnel charts. Is `cardType !== "funnel"` a valid replacement for `isScalarSeries`?